src/index.html

```diff
@@ -11,12 +11,105 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="dr"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              class="nav-link is-active"
+              href="Apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="Samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="Smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              data-qa="hover"
+              href="Laptops & Computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="Gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="Tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="Photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="Video"
```

Comment:
The `href` attribute for the 'Video' link is set to 'Video', which is not a valid URL. It should be a valid URL or path. Consider updating it to the correct destination.